docs/for-devs/console/netpols.md

```diff
@@ -4,59 +4,64 @@ title: Network Policies
 sidebar_label: Network Policies
 ---
 
-A Network Policy is a self-service feature for creating Kubernetes Network Policies (Ingress) and Istio Service Entries (Egress).
+Kubernetes Network Policies (Ingress) and Istio Service Entries (Egress) are managed via two dedicated UIs: **Inbound Rules** and **Outbound Rules**. When Ingress control is enabled for your team, _all_ pod‑to‑pod traffic is denied by default. When Egress control is enabled, _all_ external traffic is denied by default. You must explicitly create rules to open up traffic.
 
-When the Network Policies `Ingress control` option is enabled for the team, all traffic to the Pods of the Team (from other Pods within the Team and from Pods in other Teams) will be blocked by default. To allow other Pods to access your Pod you will need to create a Network Policy of type `ingress`.
-
-When the Network Policies `Egress control` option is enabled for the team, all traffic to external endpoints (outside of the cluster) is blocked by default. To allow access to external endpoints you will need to create a Network Policy of type `egress`.
-
-## Network Policies (all)
-
-All Network Policies of the Team are listed here.
+---
 
-| Property      | Description                                                     |
-| ------------- | --------------------------------------------------------------- |
-| Name          | The name of the Network Policy                                  |
-| Rule type     | Type of the Network Policy (Ingress or Egress)                  |
-| Team          | The name of the Team responsible for the build                  |
+## Inbound Rules
 
-## Create a Network Policy
+This page lists every Ingress rule for your team:
 
-1. Enter a name for the Network Policy.
+| Property | Description                                         |
+| -------- | --------------------------------------------------- |
+| Name     | The name of the Inbound rule                        |
+| Source   | The workload (and its pod label) allowed to connect |
```

Comment:
The workload network traffic that is going to be accepted by the target workload

Reply:
Although more correct, I personally find it harder to understand

Reply:
I did change it for now